usaon_vta_survey/templates/users.html

```diff
@@ -0,0 +1,29 @@
+{% extends 'base.html' %}
+
+
+{% block content %}
+  <h2>{% block title %}Users{% endblock %}</h1>
+
+  {% if current_user.role=='admin' %}
```

Comment:
Let's have some "You don't belong here!" type of message if the user isn't an admin.

My personal preference is to show the "not admin" case first, because it's really small and if it were at the end it would be a little more hidden.